### PR TITLE
add vale support to docs

### PR DESCRIPTION
**Pull request by joshmarinacci**
For linear ticket TRUNK-11859

https://linear.app/trunk/issue/TRUNK-11859/fix-typos-with-trunk-missing-capitalization

### CODE REVIEW

File: code-quality/debugging.md [hunk 1]
```diff
@@ -84,13 +84,13 @@ actions:
 
 This will disable the checks for just the current user. The `.trunk/user.yaml` file is specifically gitignored but will be loaded locally if present.
 
-## What should I do if a linter process seems to take longer than expected during a Trunk check?
+## What Should I do if a Linter Process Seems to Take Longer than Expected During a Trunk Check?
```
Comment (TylerJang27) [inline code-quality/debugging.md:87]: We should ignore this vale error.

**Review by TylerJang27 — APPROVED** [review]
LGTM. Can you do a `trunk check -a --filter=vale` and see how much it turns up? I suspect we will want to follow this up with a PR to tune the config a bit more.

File: code-quality/debugging.md (hunk 1 above)
Comment (joshmarinacci) [reply code-quality/debugging.md:87]: You mean we should ignore the Title Case rule?

Comment (TylerJang27) [reply code-quality/debugging.md:87]: I meant trunk-ignore but whatever works

**Pull request merged**